arches_her/views/resource.py

```diff
@@ -57,15 +57,24 @@ def get(self, request, resourceid=None):
                 document = se.search(index='resources', id=resourceid, doc_type='_doc')
                 datatype_factory = DataTypeFactory()
                 additional_data = {}
+                display_description = ''
+                display_name = ''
+                map_popup = ''
                 if document['_source']['graph_id'] == '8d41e49e-a250-11e9-9eab-00224800b26d':
                     tiles = build_resource_dict([resource], active_cons_node_list, datatype_factory)
                     additional_data = tiles[0]
+                if document['_source']['displaydescription'][0]['value']:
+                    display_description = document['_source']['displaydescription'][0]['value']
+                if document['_source']['displayname'][0]['value']:
+                    display_name = document['_source']['displayname'][0]['value']
+                if document['_source']['map_popup'][0]['value']:
+                    map_popup = document['_source']['map_popup'][0]['value']
```

Comment:
this gets the first display descriptor. It should get the one for the active language
![image](https://github.com/archesproject/arches-her/assets/25295526/4c227850-87f8-46f3-981e-84c22668a3b3)